syntax = "proto3";

service TinyKVPPService {
    rpc Put (PutRequest) returns (PutResponse);
    rpc Get (GetRequest) returns (GetResponse);
}

message PutRequest {
    string key = 1;
    string value = 2;
}

message PutResponse {
    string status = 1;
}

message GetRequest {
    string key = 1;
}

message GetResponse {
    string value = 1;
}